README.md

```diff
@@ -1 +1,55 @@
-Review Past project
\ No newline at end of file
+# Review Past Project
+
+> This is a code review practice to test my code reviewing skills.
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+- Git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone the repository and change the directory to blog
+
+```
+  $ git clone git@github.com:Favourezeugwa/Review-past-project.git
+
+  $ cd Review-past-project
+```
+
+### Install
+
+Install all project dependencies by running the command below
+
+### Run the application
+
+
+### Visit the homepage
+
+
+### Check linter errors
+
+
+### Authors
+
+👤 **Favour Amarachi Ezeugwa**
+
```

Comment:
How about someone who admires your website and wishes to contact you?  You might as well consider adding links to your professional social media handles such as Github, LinkedIn, and even Twitter. 

This will enable easy access for anyone who would love to connect with you. 🤝